File: Angular/src/app/inventory/interfaces/transfer.ts
import {ItemNote} from './item-note';

export class Transfer {
  constructor(
    public origin: number = 0, // Id Almacén de origen.
    public target: number = 0, // Id Almacén destino.
    public motivo: number = 0, // Motivo del Traslado.
    public startDate: any = null, // Fecha de registro.
    public itemNotes: Array<ItemNote> = new Array<ItemNote>()) {
  }
}

File: src/Nebula/Data/ViewModels/Transfer.cs
namespace Nebula.Data.ViewModels;

/// <summary>
/// Nota Transferencia entre almacenes.
/// </summary>
public class Transfer
{
    /// <summary>
    /// Id Almacén de origen.
    /// </summary>
    public int Origin { get; set; }

    /// <summary>
    /// Id Almacén destino.
    /// </summary>
    public int Target { get; set; }

    /// <summary>
    /// Motivo del Traslado.
    /// </summary>
    public int Motivo { get; set; }

    /// <summary>
    /// Fecha de registro.
    /// </summary>
    public DateTime StartDate { get; set; }

    /// <summary>
    /// Detalle de Transferencia.
    /// </summary>
    public List<ItemNote> ItemNotes { get; set; }
}
